pkg/columns/formatter/textcolumns/bridge.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package textcolumns
+
+// This satisfies our current implementation
+
+// TransformEvent is deprecated
+func (tf *TextColumnsFormatter[T]) TransformEvent(event *T) string {
+	return tf.FormatEntry(event)
+}
+
+// BuildColumnsHeader is deprecated
+func (tf *TextColumnsFormatter[T]) BuildColumnsHeader() string {
+	return tf.FormatHeader()
+}
```

Comment:
This is exactly what will replace the code of the [BaseParser](https://github.com/kinvolk/inspektor-gadget/blob/5c9e9d4db3ee7f98d9a078139c08b8bb54197797/cmd/common/utils/parser.go#L32). It is in charge of calling the `columns` package to achieve what the CLI needs while hiding all the complexity.